todo-app/src/App.js

```diff
@@ -1,52 +1,60 @@
 //jshint esnext: true
 
-import React from 'react';
-
-import TodoList from './Components/todoList';
-import AddTodoItem from './Components/AddTodoItem';
-import './App.css';
+import React from "react";
 
+import TodoList from "./Components/TodoList";
```

Comment:
You've got a compile error here, not sure how it was working on your machine. You're importing from a file that does not exist: `./Components/TodoList`. You just need to capitalise your `todoList.js` file!